src/platforms/javascript/common/performance/index.mdx

```diff
@@ -96,7 +96,7 @@ Without sampling, our atomatic instrumenation will send a transaction any time a
 
 ## Connecting Backend and Frontend Transactions
 
-To connect backend and frontend transactions into a single coherent trace, Sentry uses a `trace_id` value which is propagated between frontend and backend. Depending on the circumstance, this id is transmitted either in a request/response header or in an HTML `<meta>` tag. Linking transactions in this way makes it possible for you to navigate between them in the Sentry UI, so you can better understand how the different parts of your system are affecting each other.
+To connect backend and frontend transactions into a single coherent trace, Sentry uses a `trace_id` value which is propagated between frontend and backend. Depending on the circumstance, this id is transmitted either in a request header or in an HTML `<meta>` tag. Linking transactions in this way makes it possible for you to navigate between them in the Sentry UI, so you can better understand how the different parts of your system are affecting each other.
```

Comment:
```suggestion
To connect transactions from different software components into a single coherent trace, Sentry uses a `trace_id` value which is typically propagated via a request header. Linking transactions in this way makes it possible for you to navigate between them in the Sentry UI, so you can better understand how the different parts of your system are affecting each other.
```

The use of `trace_id` to connect transactions is not limited to what this section was originally about: the specific backend => frontend flow.

The suggestion makes it a more general introduction, leaving the details for the next two subsections.

Reply:
I like it, but in this case this page lives specifically in the browser (and eventually node) docs, so the front/back idea applies. I'm going to keep this in my back pocket, though, in case we ever try to pull this page into `/common` (which we probably will at some point).